kotlin/lib/src/main/kotlin/EventTypeListOptions.kt

```diff
@@ -0,0 +1,3 @@
+package com.svix.kotlin
+
+class EventTypeListOptions(var withContent: Boolean = false) : ListOptions()
```

Comment:
This is wrong, it means you can't set iterator/limit in EventTypeListOption, you need to put them as params and pass to ListOptions iirc.

Reply:
Yeah it was WIP, changed to builder